content/en/system_config/integration.md

```diff
@@ -0,0 +1,74 @@
+
+A package manager looking to adopt Sigstore as part of its artifact signing and verification workflows will generally follow these steps, adjusted appropriately to the nuances of the specific programming language and/or ecosystem:
+
+1. Develop language-specific implementation of Sigstore’s signing and verification workflows (if one does not already exist)
+2. Integrate language-specific Sigstore signing and verification functionality into package management tooling
+3. Add support for storing Sigstore signatures & attestations in the package registry for consumers to access
+4. Publish community RFC proposal describing end-to-end workflows, ecosystem-specific threat model, and roadmap for adoption
+5. Once RFC proposal is approved per community norms, drive implementation plan
+6. Work to drive initial package adoption strategy; this often involves highly visible or valuable packages that can serve as references for other package maintainers within the ecosystem
+7. Work with popular build & packaging extensions (e.g. GoReleaser, JReleaser) and builder templates (Jenkins plugins, GitHub Actions, 
+8. Release all code & service extensions required for Sigstore support in a fully-supported mode
+9. Presuming success to this point, set date for mandating Sigstore signing and verification for all packages in ecosystem   
+
+Integrating Sigstore with your own applications provides an effective way to enhance security:
+
+- The ability to control how Sigstore is used.
+- The creation of your own user interfaces.
+- Package integrity.
+- Key management unecessary.
+- Transparency.
+
+You can do this while using the underlying Sigstore software that already exists.
+
+There are several libraries available for developers who want to integrate Sigstore signing and/or verification into their project:
+
+- Cosign, is a legacy system that still should be used for signing, and Sigstore-go, which is recommended for use in verification. Cosign was developed with a focus on container image signing/verification and has a rich CLI and a long legacy of features and development.
+- Sigstore-go is a more minimal and friendly API for integrating Go code with Sigstore, with a focus on a unified format for Sigstore verification metadata.
+- Sigstore-python is a python language based API.
+- Sigstore-JS for code signing  NPM packages
+- Sigstore-java for Java based applications (pre-release).
+- Sigstore-rs for Rust applications (pre-release).
+
+**NOTE:** Each of the above libraries has their own entry points.  Refer to the specific documentation of each library for details. You can also get support on Slack at the "#clients" channel.
```

Comment:
I would add links to the libraries in the list above

Reply:
links added